src/mscorlib/src/System/Buffer.cs

```diff
@@ -262,6 +262,15 @@ internal unsafe static void Memmove(byte* dest, byte* src, nuint len)
         {
 #if AMD64 || (BIT32 && !ARM)
             const nuint CopyThreshold = 2048;
+#elif ARM64
+#if PLATFORM_WINDOWS
+            // TODO-ARM64-WINDOWS-OPT determine optimal value for Windows
```

Comment:
Should we instead open issues to track this TODOs? I'm not a fan of having TODOs without an issue as sometimes we don't even revisit this TODOs. With the issue we will track it and push toward doing it.

Reply:
@safern We have been doing "in addition" rather than "instead".  I'll open the issues.  Objections?

Reply:
Issues opened.

Reply:
> Issues opened.

Thanks. "In addition" is fine, as long as we have tracking issues for pending work.

Reply:
Could you please add links to the issues to the code comment?

Reply:
@jkotas Do you want full links or just issue #

Reply:
Full links are better.

Reply:
Done